### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by robinmoussu**

### CODE REVIEW

**Comment by robinmoussu** [comment]
As described in https://github.com/jtroo/kanata/issues/676 I tried to build driverkit. I had a few build issues that I fixed. Kanata works with this fork using MacOS Catalina (10.15.7).

File: c_src/driverkit.hpp [hunk 1]
```diff
@@ -67,7 +68,11 @@ void block_till_listener_init();
 void close_registered_devices();
 void notify_start_loop();
 int  init_sink();
-void exit_sink();
+#ifdef USE_KEXT
+    int exit_sink();
+#else
+    void exit_sink();
```
Comment (Psych3r) [inline c_src/driverkit.hpp:74]: How about we change the `void exit_sink()` to `int` too and have it just return 0.
I think I'd like it better to have a singe exit_sink function declared.

File: build.rs [hunk 2]
```diff
@@ -16,6 +16,7 @@ fn main() {
             build.flag("-D");
             build.flag("USE_KEXT");
             build.include("c_src/Karabiner-VirtualHIDDevice/dist/include");
+            build.include("c_src/Karabiner-DriverKit-VirtualHIDDevice/src/Client/vendor/include");
```
Comment (Psych3r) [inline build.rs:19]: I don't think this include is needed for `kext`. Can you confirm you can't compile without it?
Maybe `driver_activated` is what required it, and now that it's behind the `USE_KEXT` flag this is no longer needed?
If you can't compile without it, it means other parts of the code need to be behind the `USE_KEXT` flag too.

**Comment by robinmoussu** [comment]
I’ve done all those modifications and it’s working correctly. Feel free to squash those commits when merging this PR.

**Pull request merged**